examples/fscloud/version.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.3.0"
+  required_providers {
+    # Pin to the lowest provider version of the range defined in the main module's version.tf to ensure lowest version still works
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "1.49.0"
+    }
+    restapi = {
+      source  = "Mastercard/restapi"
+      version = ">=1.18.0"
+    }
```

Comment:
If the providers are removed, then this will not be required.

Reply:
see https://github.com/terraform-ibm-modules/terraform-ibm-icd-mongodb/pull/43/files#r1126978311